src/GuardClauses.UnitTests/GuardAgainsOutOfRange.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Ardalis.GuardClauses;
+using System;
+using Xunit;
+
+namespace GuardClauses.UnitTests
+{
+    public class GuardAgainsOutOfRange
+    {
+        [Fact]
+        public void DoesNothingGivenInRangeValueUsingShortcutMethod()
+        {
+            Guard.AgainsOutOfRange(2, "index", 1, 5);
+        }
+
+        [Fact]
+        public void DoesNothingGivenInRangeValueUsingExtensionMethod()
+        {
+            Guard.Against.OutOfRange(2, "index", 1, 5);
+        }
+
+        [Fact]
+        public void ThrowsGivenOutOfRangeValueUsingShortcutMethod()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() => Guard.AgainsOutOfRange(5, "index", 1, 4));
```

Comment:
Given that your guard clause has an || in its conditional, it's probably worth testing at least one case that exceeds and one that is below your range limits. Again you can use [Theory] for this.